--- a/src/Flow/Parquet/Dremel/ColumnData/FlatValue.php
+++ b/src/Flow/Parquet/Dremel/ColumnData/FlatValue.php
@@ -4,6 +4,7 @@
 
 namespace Flow\Parquet\Dremel\ColumnData;
 
+use Flow\Parquet\BinaryReader\Bytes;
 use Flow\Parquet\ParquetFile\Schema\FlatColumn;
 
 final readonly class FlatValue
@@ -12,7 +13,7 @@
         public FlatColumn $column,
         public int $repetitionLevel,
         public int $definitionLevel,
-        public int|float|string|bool|null $value = null,
+        public int|float|string|bool|null|Bytes $value = null,
     ) {
     }
 
